src/plugins/blocksmith/components/document-block-modal/selector.js

```diff
@@ -20,11 +20,14 @@ export default (state, { block }) => {
   const documentAssetSort = (request && request.get('sort').getValue()) || SearchAssetsSort.CREATED_AT_DESC.getValue();
   const documentAssetNodes = (response && response.get('nodes')) || [];
   const isDocumentAssetSearchFulfilled = status === STATUS_FULFILLED;
-
+  const currentPage = (request && request.get('page')) || 1;
+  const requestVal = request;
   return {
     documentAssetNodes,
     documentAssetSort,
     documentNode: block.has('node_ref') ? getNode(state, block.get('node_ref')) : null,
+    currentPage,
+    requestVal,
```

Comment:
@nanetteCodes you can just pass request, dont need to put it into a different var